### ISSUE TITLE
[Bug] No real-time updates for inbox

### ISSUE THREAD

**Issue by tprudentova** [linagora/tmail-flutter#3076]
0.11.4, tmail.linagora.com, tested on Android; Safari, Opera, Firefox

1. Send an email to email.linagora.com
2. Go to the email.linagora.com account
3. Stay on the page without updating it 
4. The inbox feed isn't updated automatically when a new letter arrives 

Expectation: 
The inbox feed is updated in real time

https://github.com/user-attachments/assets/930285a7-c0bd-4e8a-aea9-9200b9370bf3

**Comment by dab246** [comment]
IMO, it might be delayed due to sending from `GMAIL` to linagora's mail. You can test further by sending email from TB

**Comment by hoangdat** [comment]
yeah, no problem with real time update for email send from our side. Gmail still delay

**Comment by tprudentova** [comment]
https://github.com/user-attachments/assets/e2e560ca-610a-4430-83df-d518dc7cd49a

Same problem with emails sent from git and linagora.com
